gpkit/feasibility.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+
+from .variables import Variable, VectorVariable
+from .substitution import get_constants
+from .varkey import VarKey
+from .posyarray import PosyArray
+
+
+def feasibility_model(model, flavour="max", varname=None, constants=None):
+    """Returns a new GP for the closest feasible point of the current GP.
```

Comment:
I vote for returning a GP/Model. My impression is that this might also clean up some of the logic in `Model.feasibility`.

Here's an argument for not directly returning the newly-created variables: if a user wants/needs access to those, he/she should specify them directly using `varname=`. Does that make any sense?